libs/api/domains/disability-license/src/lib/disability-license.module.ts

```diff
@@ -1,9 +1,12 @@
 import { Module } from '@nestjs/common'
-import { DisabilityLicenseClientModule } from '@island.is/clients/disability-license'
+import {
+  DisabilityLicenseApiProvider,
+  DisabilityLicenseClientModule,
+} from '@island.is/clients/disability-license'
 import { DisabilityLicenseResolver } from './disability-license.resolver'
 
 @Module({
-  providers: [DisabilityLicenseResolver],
+  providers: [DisabilityLicenseResolver, DisabilityLicenseApiProvider],
```

Comment:
Modules should not provide stuff from other modules. Either it's exported or not.